src/main/java/io/temporal/internal/replay/ReplayWorkflowExecutor.java

```diff
@@ -460,7 +464,7 @@ private boolean handleWorkflowTaskImpl(
         forceCreateNewWorkflowTask =
             processEventLoop(
                 startTime,
-                startedEvent.getWorkflowTaskTimeoutSeconds(),
+                (int) Durations.toSeconds(startedEvent.getWorkflowTaskTimeout()),
```

Comment:
should we have processEventLoop take in a java.time.Duration instead?

Reply:
Yes